store/src/java/com/zimbra/cs/mailbox/CalendarItem.java

```diff
@@ -3522,7 +3522,11 @@ private void createPseudoExceptionForSingleInstanceReplyIfNecessary(Invite reply
                         if (itemIdGetter != null) {
                             /* ZWC expects a different mail item id for each exception */
                             localException.setMailItemId(itemIdGetter.get());
+                        } else {
+                            localException.setMailItemId(reply.getMailItemId());
```

Comment:
Please add ticket-id as comment for future reference

Reply:
Done